compiler/rustc_feature/src/removed.rs

```diff
@@ -148,6 +148,10 @@ declare_features! (
     (removed, const_raw_ptr_to_usize_cast, "1.55.0", Some(51910), None,
      Some("at compile-time, pointers do not have an integer value, so these casts cannot be properly supported")),
 
+    /// Allows `impl Trait` in bindings (`let`, `const`, `static`).
+    (removed, impl_trait_in_bindings, "1.55.0", Some(63065), None,
+     Some("removed due to being incomplete and unstable")),
```

Comment:
well, that is true for a lot of features 😆 

Maybe say something like "the implementation was not maintainable, the feature may get reintroduced once the current refactorings are done"